src/styles/themes/default.js

```diff
@@ -65,6 +65,7 @@ const darkTheme = {
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.greenHighlightBackground,
     dropUIBG: 'rgba(6,27,9,0.92)',
+    receiptDropUIBG: 'rgba(3, 212, 124, 0.84)',
```

Comment:
```js
function alphaToHex(number) {
    if (!_.isNumber(number) || number < 0 || number > 1) {
        throw new Error('number must be an integer between 0 and 1 (inclusive)');
    } 
    return (number * 255).toString(16).slice(0, 2);
}
```

```suggestion
    receiptDropUIBG: `${colors.green}${ColorUtils.alphaToHex(0.84)}`,
```

Reply:
Why is this preferred over RGBA?

Reply:
Because it references the colors from `colors.js` – that way if we can later adjust the color in just one place.